auth0/src/main/java/com/auth0/android/provider/WebAuthProvider.java

```diff
@@ -324,7 +411,22 @@ public void start(@NonNull Activity activity, @NonNull AuthCallback callback) {
     // Public methods
 
     /**
-     * Initialize the WebAuthProvider instance with an account. Additional settings can be configured
+     * Initialize the WebAuthProvider instance for logging out the user using an account. Additional settings can be configured
+     * in the LogoutBuilder, like setting the federated parameter.
+     *
+     * @param account to use for authentication
+     * @return a new Builder instance to customize.
+     */
+    public static LogoutBuilder clearSession(@NonNull Auth0 account) {
```

Comment:
Feedback on the method name? The comments below will get removed later.

Reply:
`logout` seems more natural. To remain consistent with authentication, what about `initLogout`?

Reply:
`clearSession` says to me that this method clears the session. But it's actually `start` that does the logout. Maybe `buildLogout` then `executeLogout`? Feels like a language convention more than anything. 

`WebAuthProvider.buildLogout(account).executeLogout(MainActivity.this, logoutCallback);`

Reply:
I like what @jimmyjames suggested to be consistent with the already existing `init` method that is used on authentication. Since both said "logout" I'll go with `initLogout` instead of `clearSession`.

Reply:
I've discussed this with some devs here. They suggested we could deprecate the existing `init` method used for authentication, and add a new one that delegates on that called `login` (just like the new SPA SDK does), and call this `logout` instead of `initLogout`. That sounds like a sweeter alternative!